spring-kafka-test/src/main/java/org/springframework/kafka/test/context/EmbeddedKafka.java

```diff
@@ -43,7 +43,7 @@
  * <p>
  * The typical usage of this annotation is like:
  * <pre class="code">
- * &#064;RunWith(SpringRunner.class)
+ * &#064;ExtendWith(SpringExtension.class)
```

Comment:
DITTO about `@SpringJUnitConfig`.
It is very rare situation when we need to use `@ExtendWith(SpringExtension.class)`.